tests/bson/bson-fromPHP-004.phpt

```diff
@@ -1,38 +0,0 @@
---TEST--
-BSON\fromPHP(): PHP documents with circular references
---FILE--
-<?php
-
-require_once __DIR__ . '/../utils/tools.php';
-
-echo "\nTesting packed array with circular reference\n";
-
-$document = ['x' => 1, 'y' => []];
-$document['y'][] = &$document['y'];
-echo toJson(fromPHP($document)), "\n";
-
-echo "\nTesting associative array with circular reference\n";
-
-$document = ['x' => 1, 'y' => []];
-$document['y']['z'] = &$document['y'];
-echo toJson(fromPHP($document)), "\n";
-
-echo "\nTesting object with circular reference\n";
-
-$document = (object) ['x' => 1, 'y' => (object) []];
-$document->y->z = &$document->y;
-echo toJson(fromPHP($document)), "\n";
-
-?>
-===DONE===
-<?php exit(0); ?>
---EXPECTF--
-Testing packed array with circular reference
-{ "x" : 1, "y" : [ [  ] ] }
-
-Testing associative array with circular reference
-{ "x" : 1, "y" : { "z" : {  } } }
-
-Testing object with circular reference
-{ "x" : 1, "y" : { "z" : {  } } }
```

Comment:
Per the [PHPC-948](https://jira.mongodb.org/browse/PHPC-948) issue description, circular references should result in exceptions. This test has been modified and moved to `bson-fromPHP_error-004.phpt`.